src/languages/_french.json

```diff
@@ -0,0 +1,196 @@
+{
+    "translation": {
```

Comment:
hey @PierrickGT! could you change the indentation of this file to `2 spaces` so that it matches all of our other language files? 

thanks!

Reply:
other than that it looks great to me! 🎉 will merge once indentation is fixed